fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -3367,8 +3367,9 @@ public ParseNode visitRevokeImpersonateBrief(StarRocksParser.RevokeImpersonateBr
 
     @Override
     public ParseNode visitGrantTablePrivBrief(StarRocksParser.GrantTablePrivBriefContext context) {
+        int objectTokenSize = context.tablePrivilegeObjectName().identifierOrStringOrStar().size();
         return visitGrantRevokePrivWithType(
-                "TABLE",
+                objectTokenSize == 1 ? "DATABASE" : "TABLE",
```

Comment:
Is this too tricky? Shall we add a new function like `visitGrantDatabasePrivBrief()`?

Reply:
I think there are too many functions in this class...I prefer not to add more unless really necessary.